src/collection.cr

```diff
@@ -63,6 +64,10 @@ module Collection(X, Y)
   end
 
   macro included
+    {% methods = @type.methods.map(&.name.stringify) %}
+    {% if !methods.includes? "dig" %}
+      include Digable
```

Comment:
It may make sense to shift this condition into `Digable` so that the methods conditionally include (or call `previous_dev` until crystal-lang/crystal#10504 is a thing). That way _all_ `Collection`'s can also be `Digable`.